pootle/core/search/backends/elasticsearch.py

```diff
@@ -80,6 +80,9 @@ def search(self, unit):
                 translation_pair = hit['_source']['source'] + hit['_source']['target']
                 if translation_pair not in counter:
                     counter[translation_pair] = 1
+                    submitted_on = None
+                    if 'submitted_on' in hit['_source']:
+                        submitted_on = hit['_source']['submitted_on']
```

Comment:
Did you try `submitted_on = hit['_source'].get('submitted_on', None)`? This would require a single-line change.